salesforce/10.11/modules/ROOT/pages/salesforce-connector-reference.adoc

```diff
@@ -3199,6 +3199,186 @@ Retrieves data from specified objects, whether or not they have been deleted.
+|===
+
+==== Output
+[%autowidth.spread]
+|===
+|Type |Array of String
+|===
+
+=== For Configurations
+* <<sfdc-config>>
+
+==== Throws
+* SALESFORCE:LIMIT_EXCEEDED
+* SALESFORCE:CONNECTIVITY
+* SALESFORCE:INVALID_RESPONSE
+* SALESFORCE:RETRY_EXHAUSTED
+* SALESFORCE:TIMEOUT
+* SALESFORCE:MUTUAL_AUTHENTICATION_FAILED
+* SALESFORCE:NOT_FOUND
+* SALESFORCE:INVALID_INPUT
+
+[[queryResultStream]]
+=== Query Result Stream (Deprecated)
+`<salesforce:query-result-stream>`
+
+
+Returns an InputStream with the query results of a submitted BatchInfo. Internally, the InputStreams contained in the sequence are requested on-demand (lazy-loading). They should be used only with query jobs.
+
+
+==== Parameters
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | Name of the configuration to use. | | x
+| Batch info a| <<BatchInfo>> |  BatchInfo to monitor. |  `#[payload]` |
+| Content type a| Enumeration, one of:
+
+** XML
+** CSV
+** JSON
+** ZIP_XML
+** ZIP_CSV
+** ZIP_JSON |  Content type used at job creation. If it is not provided, the default value is used. |ContentType.XML  |
+| Output Mime Type a| String | MIME type of the payload output of this operation.  |  |
+| Output Encoding a| String | Encoding of the payload output of this operation. |  |
+| Streaming Strategy a| * <<repeatable-in-memory-stream>>
+* <<repeatable-file-store-stream>>
+* <<non-repeatable-stream>> |  Configure how Mule processes streams with streaming strategies. Repeatable streams are the default behavior. |  |
+| Headers a| Object |  |  |
+| Read Timeout a| Number |  Specifies the amount of time, in the unit defined in *Time unit*, that the consumer waits for a response before it times out. |  |
+| Time unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  A time unit that qualifies the Read Timeout. |  |
+| Target Variable a| String | Name of a variable that stores the operation's output. |  |
+| Target Value a| String |  An expression that evaluates the operation's output. The expression outcome is stored in the target variable. |  `#[payload]` |
+| Reconnection Strategy a| * <<reconnect>>
+* <<reconnect-forever>> |  A retry strategy in case of connectivity errors. |  |
+|===
+
+==== Output
+[%autowidth.spread]
+|===
+|Type |Binary
+|===
+
+=== For Configurations
+* <<sfdc-config>>
+
+==== Throws
+* SALESFORCE:LIMIT_EXCEEDED
+* SALESFORCE:CONNECTIVITY
+* SALESFORCE:INVALID_RESPONSE
+* SALESFORCE:RETRY_EXHAUSTED
+* SALESFORCE:TIMEOUT
+* SALESFORCE:MUTUAL_AUTHENTICATION_FAILED
+* SALESFORCE:NOT_FOUND
+* SALESFORCE:INVALID_INPUT
+
+[[queryResultStreamById]]
+=== Query Result Stream By Id
+`<salesforce:query-result-stream-by-id>`
+
+
+Returns an InputStream with the query results for a specific result ID of a submitted BatchInfo. You will need to obtain the result IDs first by using the <<queryResultList>> operation.
```

Comment:
"You will need to obtain" > "Obtain the result IDs first..." or "You must obtain the result IDs first.."

Reply:
As you know, we are trying to use the operation names that appear in Studio wherever we can, so <<queryResultList>> would be *Query Result List*.